backend/accounts.go

```diff
@@ -1196,7 +1196,7 @@ outer:
 // a user-facing setting. Now we simply use it for migration to decide which coins to add by
 // default.
 func (backend *Backend) persistDefaultAccountConfigs(keystore keystore.Keystore, accountsConfig *config.AccountsConfig) error {
-	if backend.arguments.Testing() {
+	if backend.Testing() {
```

Comment:
I replaced all occurrences of `backend.arguments.Testing()` with `backend.Testing`. Prior to this PR the two calls are the same (`backend.Testing()` calls `backend.arguments.Testing()`) but with this new PR there is more logic in `backend.Testing()` - The alternative would have be to change the field `arguments.testing` but since everything in arguments is unexported I prefer not to